src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps;
+ 
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.util.List;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+ 
+public final class QuizTimingPropertiesUtils {
+ 
+    //This function gets the the "quiz_timestamp" property of the entity that is fed into the function
+    public Object getTimestampProperty(String entity, DatastoreService datastore) {
+        try {
+            Query query = new Query(entity);
+            PreparedQuery pq = datastore.prepare(query);
+ 
+            try {
+                Entity fetched_item = pq.asList(FetchOptions.Builder.withLimit(1)).get(0);
+                Object timestamp_of_fetcheditem = fetched_item.getProperty("quiz_timestamp");
```

Comment:
What specific type is timestamp_of_fetcheditem? We want to be more specific than using the generic "Object" class.

Reply:
I got rid of the timestamp_of_fetcheditem variable like you suggested, should I still be more specific about the object I am returning? I just do it this way because it is easy to then convert to a string later

Reply:
I do not, when I print the type it says it is a "java.lang.Object"

Reply:
https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/PropertyContainer.html#getProperty-java.lang.String-

Well this is really annoying but I guess it does just return Object... That's... just the worst... but nothing you can do about that, so you're good to keep everything as "Object" lol. Sorry about the non-relevant advice!